Makefile

```
@@ -0,0 +1,11 @@
+migrate:
+	docker-compose exec web ./manage.py migrate
+
+createsuperuser:
+	docker-compose exec web ./manage.py createsuperuser
+
+collectstatic:
+	docker-compose exec web ./manage.py collectstatic
+
+runtests:
+	docker-compose exec web pytest
```

O `docker-compose exec` não depende de existir um instância dessa imagem em execução?

Quais os prós e contra de termos `docker-compose run --rm web …`?

```suggestion
	docker-compose run --rm web ./manage.py migrate

createsuperuser:
	docker-compose run --rm web ./manage.py createsuperuser

collectstatic:
	docker-compose run --rm web ./manage.py collectstatic

runtests:
	docker-compose run --rm web pytest
```

---

Essa eu peguei do outro PR, mas pra mim faz sentido sim!